tests/group/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required (VERSION 3.0) # The minimum version of CMake necessary to build this project
+project (group) # The name of our project
+
+declare_trisycl_test(TARGET group TEST_REGEX
+" 10
+ 2
+ 0
+ 5
+5 
```

Comment:
They are needed. The regex language is whitespace and new line sensitive. If I wanted to allow spaces anywhere, I'd need wildchar (*) for the space character EVERYWHERE which would make the regex very noisy.

It bugged me very much when I created the MS Unit Testing Framework code for the hand-crafted Solution files that there are a lot of trailing spaces from the test outputs. Without wanting to change the source code of many of the tests, I just played along.

ps.: It was very hard to find all the trailing whitespaces, because cmd.exe, the Windows terminal swallows them and I could not even copy+paste the test output from console if it seemed to be in accordance with the LIT comments. I had to spin up VS debugger windows which preserved trailing spaces.